apidef/oas/middleware.go

```diff
@@ -1274,3 +1274,32 @@ func (e EndpointPostPlugins) ExtractTo(meta *apidef.GoPluginMeta) {
 	meta.PluginPath = e[0].Path
 	meta.SymbolName = e[0].Name
 }
+
+type CircuitBreaker struct {
```

Comment:
shall we add the corresponding Tyk classic api definition configuration in godoc sytle like https://github.com/TykTechnologies/tyk/blob/354bb28accd301f9d05d9fb36ca5064dc52e338b/apidef/oas/server.go#L14

Reply:
Added requested changes.